src/main/java/com/acmcsuf/bot_committee/Register.java

```diff
@@ -87,6 +96,7 @@ private List<CommandData> guildCommands()
                 new SubcommandData( com.acmcsuf.bot_committee.Commands.TRIGGER_DELETE,
                         com.acmcsuf.bot_committee.Commands.TRIGGER_DELETE_DESCRIPTION )
                         .addOption( OptionType.STRING, "word", "Trigger word", true, true );
```

Comment:
For consistency, shall we also pass values from the Commands constants class to `addOption`?